crates/net/eth-wire/src/types/blocks.rs

```diff
@@ -71,13 +74,15 @@ impl From<Vec<H256>> for GetBlockBodies {
 // TODO(onbjerg): We should have this type in primitives
 /// A response to [`GetBlockBodies`], containing bodies if any bodies were found.
 #[derive_arbitrary(rlp, 10)]
-#[derive(Clone, Debug, PartialEq, Eq, RlpEncodable, RlpDecodable, Default)]
```

Comment:
Can't we still derive this with the new RLP macro you merged for trailing opts?

Reply:
yup, this was the motivation for that PR